src/main/java/com/dawkastresu/medical_clinic/Patient.java

```diff
@@ -0,0 +1,15 @@
+package com.dawkastresu.medical_clinic;
+
+import lombok.Data;
+
+import java.time.LocalDate;
+@Data
+public class Patient {
+    private String email;
+    private String password;
+    private long idCardNo;
```

Comment:
Czy numer dowodu osobistego powinien byc longiem -> typem liczbowym? Nie powinien byc, nie jest to dana na ktorej ty bedizesz wykonywal jakies operacje matematyczne jak dodawanie, mnozenie itp. To jets dana reprezentujaca numre dowodu osobistego